### PR TITLE
Implements multiple outputs to account for package name change (close…

### LINKED ISSUE #3: Package name was changed on PyPI to pytest-check
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

The name of this package has changed from `pytest_check` to `pytest-check`.  The PyPI links would not work anymore.

It would be good if this feedstock could catch-up.

### Installed packages

```shell
Not relevant
```

### Environment info

```shell
Not relevant
```

### PR DESCRIPTION
**Pull request by anjos**
…s #3)

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This fixes #3 by providing a second "meta-package", with the legacy name (ie. `pytest_check`), that solely depends on `pytest-check` itself.  The user may install any and the installation should just work (tm).

### CODE REVIEW

**Comment by anjos** [comment]
@beckermr: can you please review this and confirm this is as suggested on #3?

**Comment by anjos** [comment]
There is one weird problem with the recipe as is tho: the ``{{ PYTHON }}`` variable on the build script entry stopped working.  Do you have any idea of what would be wrong?

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
I think for outputs you may need a separate script.

**Comment by beckermr** [comment]
Sorry that was not clear. A separate build script. Don't name it build.sh.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,40 +10,63 @@ source:
...
-    - pip check
-    - pytest
+  number: 1
+
+outputs:
+  - name: {{ name }}
+
+    build:
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv
```
Comment (beckermr) [inline recipe/meta.yaml]:
```suggestion
      script: ${PYTHON} -m pip install . -vv
```

**Comment by beckermr** [comment]
I wonder if this will work.

**Comment by beckermr** [comment]
Something isn't right. There is an extensive clobber warning for files in the _pytest folder in site packages.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,40 +10,63 @@ source:
...
-    - pip check
-    - pytest
+  number: 1
+
+outputs:
+  - name: {{ name }}
+
+    build:
+      noarch: python
+      script: ${PYTHON} -m pip install . -vv
```
Comment (beckermr) [inline recipe/meta.yaml]:
```suggestion
      script: ${PYTHON} -m pip install . -vv --no-deps
```

**Review by beckermr — CHANGES_REQUESTED** [review]
Something is still not right. The build is installing flit-core when it should already be in host.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,40 +10,63 @@ source:
...
-    - pip check
-    - pytest
+  number: 1
+
+outputs:
+  - name: {{ name }}
+
+    build:
+      noarch: python
+      script: ${PYTHON} -m pip install . -vv --no-deps
```
Comment (beckermr) [inline recipe/meta.yaml]:
```suggestion
      script: ${PYTHON} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -10,40 +10,63 @@ source:
...
+        - pytest_check
+      requires:
+        - pip
+      source_files:
+        - tests/
+        - examples/
+        - tox.ini
+      commands:
+        - pip check
+        - pytest
```
Comment (beckermr) [inline recipe/meta.yaml:43]: These tests are not being run.

**Review by beckermr — CHANGES_REQUESTED** [review]

**Review by beckermr — APPROVED** [review]

**Comment by anjos** [comment]
Many thanks, @beckermr!

**Pull request merged**